spec/requests/admin/users_controller_spec.rb

```diff
@@ -898,6 +899,15 @@
       expect(reg_user).to be_silenced
     end
 
+    it "will create a silenced record" do
+      put "/admin/users/#{reg_user.id}/silence.json"
+
+      expect(response.status).to eq(200)
+      reg_user.reload
+      expect(reg_user).to be_silenced
+      expect(reg_user.silenced_record).to be_present
```

Comment:
We don't actually need a whole new test to check a `silenced_record` was created. This line could be added to the previous "punishes the user for spamming" test.

Reply:
done